cve_bin_tool/output_engine/console.py

```diff
@@ -174,6 +174,23 @@ def _output_console_nowrap(
                 break
 
         # Show table of vulnerable products mapped to filename paths
+        # As path names can be long, these maybe replaced with a note which
```

Comment:
```suggestion
        # As path names can be long, these may be replaced with a note which
```

Reply:
@ffontaine @terriko The test PDF data in test_output_engine.py (PDF_OUTPUT) needs to be modified slightly (no idea why pdftotext puts the new columns at the end of the list) as follows:

`

        "4. List of Vulnerabilities mapped to Components"
        "The following vulnerabilities are reported against the identified components which"
        "contain the reported libraries."
        "Vendor"
        "Product"
        "Version"
        "vendor0"
        "product0"
        "1.0"
        "vendor0"
        "product0"
        "2.8.6"
        "vendor1"
        "product1"
        "3.2.1.0"
        "Page 2"
        "Root"
        "Filename"


`

Reply:
@ffontaine Need to also make correction to output_engine/__init__.py to handle long root/filenames in PDF documents

Update lines 338-343 as follows:

`

            pdfdoc.createtable(
                "Applicationlist",
                ["Vendor", "Product", "Version", "Root", "Filename"],
                pdfdoc.tblStyle,
                [10, 10, 10, 15, 15],
            )


`